fe/fe-core/src/main/java/org/apache/doris/task/HadoopLoadPendingTask.java

```diff
@@ -70,41 +70,41 @@ public HadoopLoadPendingTask(LoadJob job) {
 
     @Override
     protected void createEtlRequest() throws Exception {
-        db.readLock();
-        try {
-            EtlTaskConf taskConf = new EtlTaskConf();
-            // output path
-            taskConf.setOutputPath(getOutputPath());
-            // output file pattern
-            taskConf.setOutputFilePattern(job.getLabel() + ".%(table)s.%(view)s.%(bucket)s");
-            // tables (partitions)
-            Map<String, EtlPartitionConf> etlPartitions = createEtlPartitions();
-            Preconditions.checkNotNull(etlPartitions);
-            taskConf.setEtlPartitions(etlPartitions);
+        EtlTaskConf taskConf = new EtlTaskConf();
```

Comment:
Table's lock should be held to call method like `createEtlPartitions()`

Reply:
add table lock in createEtlPartitions